ci-build.py

```diff
@@ -46,16 +45,14 @@
 build_env = os.environ.copy()
 build_env.update({
     'CC': args.cc,
-    'CXX': args.cxx,
-    })
+    'CXX': args.cxx})
```

Comment:
Is this really how we want it to be? I usually try to format all lines in a listing like this the same (hence the trailing comma even on the last entry), and I assumed this to be the generally preferred way™.

Did flake8 really suggest this change? I can't reproduce it in my environment.

Reply:
That was one question I asked further down in this PR.
The other solution is having the closing bracket on a separate line indented with the beginning of the line opening the bracket, like @The-Compiler suggested.
I will adjust this tonight.

Reply:
Please take a look at the way I did it now and tell me if that looks better

Reply:
Yup, much better. I don't have a strong opinion on how to indent the closing brackets, as long as they're on a separate line. To my surprise, flake8 doesn't seem to care either.

My slight preference would be to indent them just like the lines before (i.e., not having them on the same level as the opening line). But I really don't mind either way.